benefits/core/templates/core/includes/form.html

```diff
@@ -3,41 +3,39 @@
 
   {% url form.action_url as form_action %}
 
-  <form id="{{ form.id }}" action="{{ form_action }}" method="{{ form.method | default:"post" | upper }}" role="form">
+  <form id="{{ form.id }}"
+        class="{{ form.classes }}"
+        action="{{ form_action }}"
+        method="{{ form.method | default:"post" | upper }}"
+        role="form">
```

Comment:
I was running the Axe devtools on form pages and noticed a very minor issue with this ARIA `role="form"` attribute.

![image](https://github.com/user-attachments/assets/5db680f1-46fd-4205-927b-853e161b1c54)

See https://www.w3.org/TR/html-aria/#el-form:

> Implicit ARIA semantics (explicitly assigning these in markup is NOT RECOMMENDED)

The implicit ARIA semantics for the `<form>` element is `role=form`, so it should _not_ be set explicitly.

Reply:
Ooh! Thank you for this!

Reply:
Will be taken care of in this PR https://github.com/cal-itp/benefits/pull/2613